posts/inside-rust/2022-02-22-compiler-team-ambitions-2022.md

```diff
@@ -0,0 +1,398 @@
+
+As you read the document, it is useful to keep in mind that [Rust is not a company][mara-post]: The teams, and the leaders of the teams, do not establish goals in a top-down manner, nor do they hand out tasks in a round-robin fashion. Instead, we collectively (and iteratively) refine our a shared vision for the future, and take steps that hopefully move towards that future. Each contributor decides for themself how much time they can afford to contribute, and that can vary wildly between contributors. The goals that we set for the project must be aligned with the goals of our current and future contributors; otherwise, they just won't get done. We have processes (e.g. [RFCs](https://github.com/rust-lang/rfcs#readme), [MCPs](https://forge.rust-lang.org/compiler/mcp.html)) that try to ensure alignment; in some ways, a document like this one is just another tool for recalibrating alignment.
+
+<!--
+But the flip side of this is: if something really is important, then there almost certainly exists a contributor willing to work on it. The real hurdle then is *enabling* that contributor to succeed.
+(Note: this is hard! Its not just about mentorship/education; its just as much about achieving *alignment* amongst the whole group!)
+-->
+
+[mara-post]: https://blog.m-ou.se/rust-is-not-a-company/
+
+[antoyo]: https://github.com/antoyo
+<!-- [antoyo zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/404242-user404242 --> <!-- @**antoyo** -->
+<!-- antoyo sponsorship: https://github.com/sponsors/antoyo -->
+[Aaron Hill]: https://github.com/Aaron1011
+<!-- [Aaron Hill zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/116083-user116083 --> <!-- @**Aaron Hill** -->
+<!-- Aaron1011: no affiliation -->
+[bjorn3]: https://github.com/bjorn3
+<!-- [bjorn3 zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/133247-user133247 --> <!-- @**bjorn3**  -->
+<!-- bjorn3 donation page: https://liberapay.com/bjorn3 -->
+[cjgillot]: https://github.com/cjgillot
+<!-- [cjgillot zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/248906-user248906 --> <!-- @**cjgillot**  -->
+<!-- no response from cjgillot re affiliation yet -->
+[davidtwco]: https://github.com/davidtwco
+<!-- [davidtwco zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/248906-user248906 --> <!-- @**davidtwco**  -->
+<!-- davidtwco affiliation: "Huawei R&D UK"-->
+[estebank]: https://github.com/estebank
+<!-- [estebank zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/119031-user119031 --> <!-- @**Esteban Küber** -->
+<!-- estebank affiliation: AWS -->
+[lcnr]: https://github.com/lcnr
+<!-- [lcnr zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/216206-user216206 --> <!-- @**lcnr** -->
+<!-- lcnr sponsorship: https://lcnr.de/funding/ -->
+[michaelwoerister]: https://github.com/michaelwoerister
+<!-- [mw zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/124287-user124287 --> <!-- @**michaelwoerister** -->
```

Comment:
```suggestion
<!-- [michaelwoerister zulip PM]: https://rust-lang.zulipchat.com/#narrow/pm-with/124287-user124287 --> <!-- @**mw** -->
```

(The idea here was to put the zulip user names in the comment, for ease of lookup/verification when adding new references to the person elsewhere.)